### DISCUSSION TITLE
Add an optional parameter to the `Point` constructor to support `lng,lat`

### DISCUSSION THREAD

**Discussion by xyy94813** [geopy/geopy#565] [Ideas]
Some geo RFC defines the point in a way that is `lng,lat`.
Such as [GeoJSON](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.1)

Is it possible to provide some way to more easily support `lng,lat`

----------

Mark: https://github.com/apache/superset/issues/19025#issuecomment-1060302269

**Comment by KostyaEsmukov** [comment]
The Point's order of coordinates is lat-lon-alt: https://geopy.readthedocs.io/en/latest/#geopy.point.Point

However, there's a little helper function, which creates a Point instance from a lon-lat-alt order: https://geopy.readthedocs.io/en/latest/#geopy.distance.lonlat

**Reply by xyy94813** [reply]
Is it support `lonlat("112, 13")` or `lonlat("-81.0 W 41.5 N")`?

**Reply by xyy94813** [reply]
I need to parse a lonlat string, the function is ineffective for me.

I write a new class inherite from [geopy.point.Point](https://geopy.readthedocs.io/en/latest/#geopy.point.Point).
Can you help me to review it? 

This is the [PR](https://github.com/apache/superset/pull/26180/files). （File is superset/utils/geopy_extensions.py）
